providers/google/tests/unit/google/cloud/transfers/test_s3_to_gcs.py

```diff
@@ -306,14 +302,12 @@ def test_get_openlineage_facets_on_start(
 class TestS3ToGoogleCloudStorageOperatorDeferrable:
     @mock.patch("airflow.providers.google.cloud.transfers.s3_to_gcs.CloudDataTransferServiceHook")
     @mock.patch("airflow.providers.google.cloud.transfers.s3_to_gcs.S3Hook")
-    @mock.patch("airflow.providers.amazon.aws.operators.s3.S3Hook")
     @mock.patch("airflow.providers.google.cloud.transfers.s3_to_gcs.GCSHook")
-    def test_execute_deferrable(self, mock_gcs_hook, mock_s3_super_hook, mock_s3_hook, mock_transfer_hook):
+    def test_execute_deferrable(self, mock_gcs_hook, mock_s3_hook, mock_transfer_hook):
         mock_gcs_hook.return_value.project_id = PROJECT_ID
 
-        mock_list_keys = mock.MagicMock()
-        mock_list_keys.return_value = MOCK_FILES
-        mock_s3_super_hook.return_value.list_keys = mock_list_keys
+        mock_s3_super_hook = mock.MagicMock()
+        mock_s3_super_hook.list_keys.return_value = MOCK_FILES
```

Comment:
This gets repeated a handful of times, we should consider moving it to a fixture in the future.

Reply:
Yeah, that could be a good future improvement. It's not all the tests but there's enough of them to maybe warrant that in the future.